### ISSUE TITLE
ARE GUI and GUI widgets not accessible

### ISSUE THREAD

**Issue by deinhofer** [asterics/AsTeRICS#126]
The ARE GUI main window and all its widgets (e.g. CellBoard, ButtonGrid, Oscilloscope,...) are not accessible.
Main problems are

  * keyboard navigable: The widgets and the subelements of the widget (e.g. CellBoard cells) should be traversable via Tab/Shift Tab.
  * A general shortcut concept should be implemented: e.g. Why not automatically assigning a numbering scheme to the GUI widgets of a model: e.g. pressing 1 1 would trigger the first cell of cellboard 1. Additionally the shortcuts could be globally accessible through a unique prefix key combination.
  * Make the contents of the GUI widgets screenreader capable: Currently the screenreader does not read out the contents of a cell.
